website/blog/2023-03-10-databend-weekly-84.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "This Week in Databend #84"
+date: 2023-03-10
+slug: 2023-03-10-databend-weekly
+tags: [databend, weekly]
+description: "Learn about the latest updates of Databend in a week!"
+contributors:
+  - name: andylokandy
+  - name: ariesdevil
+  - name: b41sh
+  - name: BohuTANG
+  - name: Carlosfengv
+  - name: Chasen-Zhang
+  - name: dantengsky
+  - name: drmingdrmer
+  - name: dusx1981
+  - name: everpcpc
+  - name: jun0315
+  - name: leiysky
+  - name: mergify[bot]
+  - name: PsiACE
+  - name: quaxquax
+  - name: RinChanNOWWW
+  - name: soyeric128
+  - name: sundy-li
+  - name: TCeason
+  - name: wubx
+  - name: xinlifoobar
+  - name: Xuanwo
+  - name: xudong963
+  - name: youngsofun
+  - name: zhang2014
+authors:
+  - name: PsiACE
+    url: https://github.com/psiace
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+> :loudspeaker: This week we announced the official release of Databend v1.0. This is an important milestone, to learn more, please check out our blog: https://databend.rs/blog/databend-release-v1.0
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+### SQL: `REPLACE INTO`
+
+The `REPLACE INTO` statement is a powerful way to insert or update data in Databend. It allows you to specify a conflict key (not necessarily a primary key) that determines whether a new row should be inserted or an existing row should be updated.
+
+If a row with the same conflict key already exists in the table, it will be replaced by the new data. Otherwise, a new row will be added with the new data. You can use this statement to easily sync data from different sources or handle duplicate records.
```

Comment:
If a row with the same conflict key already exists in the table, Databend will update the row with the new data. Otherwise, the new data will be added to the table as a new row.